interview-questions.md

```diff
@@ -4,47 +4,47 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: It is a method that takes in any number of state objects and allows them to be reffered to by shorthand for the duration of the current method you're in.
 
-  Researched answer:
+  Researched answer: After doing some research I discovered that destructing is a JavaScript feature that allows you to extract multiple pieces of data from an array or object and assign them to their own variables
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event occurs when code allows a user input to update the DOM
 
-  Researched answer:
+  Researched answer: // Had some trouble finding an answer that made sense to me //
 
 
 
 3. What is the difference between a class and an object?
 
-  Your answer:
+  Your answer: A class is a blueprint for which objects are created on
 
-  Researched answer:
+  Researched answer: Looking into it a little more I discovered that a class is a template for objects. A class defines object properties including a valid range of values, and a default value. A class also describes object behavior. An object is a member or an "instance" of a class. An object has a state in which all of its properties have values that you either explicitly define or that are defined by default settings.
 
 
 
 4. What is the difference between a HTML div and a span?
 
-  Your answer:
+  Your answer: I haven't leanred about a span in HTML but a div is a basic container where text can be added and targeted. 
 
-  Researched answer:
+  Researched answer: I searched the internet and found that they are actually pretty similar. According to htmldog.com "The difference between span and div is that a span element is in-line and usually used for a small chunk of HTML inside a line (such as inside a paragraph) whereas a div (division) element is block-line (which is basically equivalent to having a line-break before and after it) and used to group larger chunks of code."
 
 
 
 5. What is React.Fragment (or <>) and why would you want to use it?
 
-  Your answer:
+  Your answer: Honestly not very sure about React.Fragment. I know it used in the render of components and I think it interacts with the html. 
 
-  Researched answer:
+  Researched answer: After looking at the dev.to website I learned that "React fragments let you group a list of children without adding extra nodes to the DOM because fragments are not rendered to the DOM. So basically we use React.Fragment where we would normally use a wrapper div."
 
 
 
 6. What are three options for creating responsive design?
 
-  Your answer:
+  Your answer: 
```

Comment:
I know you also mentioned you needed more time here. Let us know what we can do to make sure you're provided enough time to complete these assessments.